### PR TITLE
Fixes to the Nebula LLM Integration

### PR DESCRIPTION
**Pull request by davidvonthenen**
This addresses some issues with introducing the Nebula LLM to LangChain in this PR:
https://github.com/langchain-ai/langchain/pull/8876

This fixes the following:
- Removes `SYMBLAI` from variable names
- Fixes bug with `Bearer` for the API KEY

Thanks again in advance for your help!
cc: @hwchase17, @baskaryan

### CODE REVIEW

File: libs/langchain/langchain/llms/symblai_nebula.py [hunk 1]
```diff
@@ -69,19 +56,19 @@ class Config:
     def validate_environment(cls, values: Dict) -> Dict:
         """Validate that api key and python package exists in environment."""
         nebula_service_url = get_from_dict_or_env(
-            values, "nebula_service_url", "SYMBLAI_NEBULA_SERVICE_URL"
+            values, "nebula_service_url", "NEBULA_SERVICE_URL"
```
Comment (baskaryan) [inline libs/langchain/langchain/llms/symblai_nebula.py:59]: you can pass a default value as a fourth param here (so you don't have to manually do it later

Comment (dvonthenen) [reply libs/langchain/langchain/llms/symblai_nebula.py]: changes made!

**Comment by dvonthenen** [comment]
fixed the formatting... should be good now

**Comment by baskaryan** [comment]
thanks @dvonthenen!

**Pull request merged**